server.js

```diff
@@ -117,6 +117,21 @@ app.post('/api/delete_record', function(req, res){
   })
 })
 
+// route for pledge
+app.post('/api/make_pledge', function(req, res){
```

Comment:
It looks like there are conflicts with thise. So resolve. Also write tests for it (perhaps ask Nick or Tom about how to do so) and refactor it using MVC (Perhaps ask Corrine about that)